### PR TITLE
Update copyright headers

### PR DESCRIPTION
**Pull request by aprokop**
Merge after #451.

### CODE REVIEW

File: test/ArborX_EnableDeviceTypes.hpp.in [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+
 #ifndef ARBORX_ENABLE_DEVICE_TYPES_HPP
 #define ARBORX_ENABLE_DEVICE_TYPES_HPP
 
 #include <Kokkos_Core.hpp>
 
 #include <tuple>
 
-using ARBORX_DEVICE_TYPES = std::tuple<@ARBORX_DEVICE_TYPES@>;
+using ARBORX_DEVICE_TYPES = std::tuple<@ARBORX_DEVICE_TYPES @>;
```
Comment (dalg24) [inline test/ArborX_EnableDeviceTypes.hpp.in:19]: Why did you add a space here

Comment (aprokop) [reply test/ArborX_EnableDeviceTypes.hpp.in]: For some reason, my vim clang-format plugin does that sometimes. Fixed.

**Comment by aprokop** [comment]
> You want to go with 2021 everywhere despite not all the files might have been touched this year already?

Yes. I don't think it matters whether a file has been touched this year or not. My thinking was that copyright was for the code, not individual files.

**Review by dalg24 — APPROVED** [review]

**Review by masterleinad — APPROVED** [review]
Fine with me.
